docs/source/tutorial/production.md

```diff
@@ -63,15 +63,3 @@ Publishing your schema to Apollo Engine unlocks many features necessary for runn
 We also want to be transparent that the features we just described, such as viewing specific execution traces and validating schema changes against recent operations, are only available on a paid plan. Individual developers just getting started with GraphQL probably don't need these features, but they become incredibly valuable as you're working on a team. Additionally, layering these paid features on top of our free developer tools like Apollo VSCode makes them more intelligent over time.
 
 We're committed to helping you succeed in building and running an Apollo graph API. This is why features such as publishing and downloading schemas from the registry, our open source offerings like Apollo Client and Apollo Server, and certain developer tools like Apollo VSCode and Apollo DevTools will always be free forever.
-
-## Deploy your graph API
-
-To deploy our app to Now, run the `now` command from the `server` directory of the app. The command may prompt you to login if you haven't already.
-
-```bash
-$ npx now
```

Comment:
According to [this comment](https://github.com/apollographql/apollo/issues/264#issuecomment-502449102), whilst `npx now` is suffering from problems, running `now` globally works fine.

So perhaps if we remove the `npx` prefix, this section can work.

(We might want to remind the reader that `npm install -g now` is needed, and let them know that `npx` may not work as expected.)